### DISCUSSION TITLE
QUML player getting stuck if student click on previos option

### DISCUSSION THREAD

**Discussion by snehal0904** [sunbird-specs/QuML#12] [General]
Hello Everyone,
We need urgent help with the QUML player.
We are using a QUML player in our application. It was working fine when we were using it.allowSkip = Yes. But in our current exams, we have to disable the skip option. Now that we have disabled the skip, we are getting issues as the student goes on next and comes to the previous option. The student is not able to go ahead. Students are getting stuck and not able to complete exams.
Example - If an exam has 10 questions, I am on the 5th question and want to go to the 2nd question to confirm the answer and go 
next but unless and until I change the answer option of the 2nd question I am not able to go ahead.

Is this issue already fixed in any version? Please share any input if anyone has any. 

@parthlawate @parthlawate @vinod303 @coolbung

**Comment by sajeshkayyath** [comment]
@snehal0904 - we will have a check on this. Can you help us us with which version of QuML player you are facing this issue?

**Reply by sajeshkayyath** [reply]
`Now that we have disabled the skip, we are getting issues as the student goes on next and comes to the previous option. The student is not able to go ahead. `

@snehal0904 - As a workaround, the navigation in the right side can be used to navigate between questions.  

![image](https://github.com/sunbird-specs/QuML/assets/124142090/e4151a3f-ffba-461d-ab61-7a3714b4b28c)

In the meantime, we will try to recreate the problem and see what is the potential fix. 

Do let us know the exact version of QuML player where you are having this issue.

**Reply by snehal0904** [reply]
We are using the old version where the web component was created in QUML. Versioning has been introduced for the last 8 months. We don't have an updated web component, but if we get the fix we will plan for an upgrade and start using the latest one.

For the navigator, As it was allowing skip, we have disabled navigation

**Reply by rajnishdargan** [reply]
@snehal0904 
As discussed we have shared the web component build offline with you by applying the fix on `release-5.7.0`
Can you please provide update if it's working fine?

Also we want to know the player's specific release to which this patch should be applied so we can apply the patch to that release and merged to all the further release versions.
cc: @sajeshkayyath @parthlawate @vinod303 @coolbung @PoojaJManjooran
